install/homebox-install.sh

```diff
@@ -29,6 +29,10 @@ chmod +x /opt/homebox
 echo "${RELEASE}" >"/opt/${APPLICATION}_version.txt"
 msg_ok "Installed Homebox"
 
+msg_info "Creating Blank ENV File"
+touch /opt/.env
```

Comment:
```bash
cat <<EOF >/opt/.env
# Application mode
# HBOX_MODE=production  # Can be one of: development, production

# Web server settings
HBOX_WEB_PORT=7745  # Port to run the web server on
HBOX_WEB_HOST=0.0.0.0  # Host to run the web server on (use 0.0.0.0 for all interfaces)

# File upload settings
HBOX_WEB_MAX_UPLOAD_SIZE=10  # Maximum file upload size supported in MB

# HTTP timeout settings
HBOX_WEB_READ_TIMEOUT=10  # Read timeout of HTTP server (seconds)
HBOX_WEB_WRITE_TIMEOUT=10  # Write timeout of HTTP server (seconds)
HBOX_WEB_IDLE_TIMEOUT=30  # Idle timeout of HTTP server (seconds)

# Storage settings
HBOX_STORAGE_DATA=/data/  # Path to the data directory
HBOX_STORAGE_SQLITE_URL=/data/homebox.db?_fk=1  # SQLite database URL

# Logging settings
HBOX_LOG_LEVEL=info  # Log level (trace, debug, info, warn, error, critical)
HBOX_LOG_FORMAT=text  # Log format (text or json)

# Mailer settings
# HBOX_MAILER_HOST=email-smtp.example.com  # Email host to use
# HBOX_MAILER_PORT=587  # Email port to use
# HBOX_MAILER_USERNAME=your_email@example.com  # Email user to use
# HBOX_MAILER_PASSWORD=your_password  # Email password to use
# HBOX_MAILER_FROM=your_email@example.com  # Email from address to use

# Swagger settings
HBOX_SWAGGER_HOST=localhost:7745  # Swagger host to use
HBOX_SWAGGER_SCHEME=http  # Swagger schema to use (http or https)

# Application options
HBOX_OPTIONS_ALLOW_REGISTRATION=true  # Allow users to register themselves
HBOX_OPTIONS_AUTO_INCREMENT_ASSET_ID=true  # Auto increment asset_id for new items

# Debug settings
# HBOX_DEBUG_ENABLED=false  # Enable or disable debugging
# HBOX_DEBUG_PORT=4000  # Debug port

# Demo settings
# HBOX_DEMO=true  # Enable demo mode (optional)
EOF

Reply:
need to test, because i dont know all correct .env's f.e. Storage & SQLite

Reply:
They are documented here https://homebox.software/en/configure-homebox.html
I have also tested them all and they update correctly.

I didn't include anything in the env file as I thought it would have made the install script long

Reply:
and which one do you need? 
you say it makes the install long, but you press all into the service? ^^

Reply:
you need both its in the documentation...
i have also mention why I pass all the info like that into the service